pytest/tests/sanity/backward_compatible.py

```diff
@@ -31,6 +32,20 @@ def main():
         "--home=%s" % node_root, "testnet", "--v", "2", "--prefix", "test"
     ])
 
+    with open('/tmp/near/backward/test0/genesis.json') as f:
+        stable_genesis = json.load(f)
+        stable_protocol_version = stable_genesis['protocol_version']
+
+    with open(
+            os.path.join(os.path.dirname(__file__),
+                         '../../../core/primitives/src/version.rs')) as f:
+        for line in f:
+            if line.startswith('pub const PROTOCOL_VERSION'):
+                current_protocol_version = int(line.split('=')[1].split(';')[0])
```

Comment:
why don't we read from `neard/res/genesis.json`?

Reply:
Good idea